### DISCUSSION TITLE
Interactive scorecard user meeting November 9, 2022

### DISCUSSION THREAD

**Discussion by randytpierce** [NOAA-GSL/MATS#836] [Scorecard - updates]
#### Navigation: The status is reachable from the front page of the scorecard app in two ways.
1) Click the green "Scorecard Status" button that is next to the "Add Row" button.
2) click the green "Submit Scorecard" button which is available after adding one or more Row Groups.

Once you are on the status display, you can return to the main page by clicking the blue "Back" button at the top left of the page.

### Usage

For the moment this data is "dummy" data and serves to illustrate the mechanisms of the status panel.

The status panel starts with a list of users which you are authorized to view their scorecards. (authorization is not currently implemented so you can see all the users.) 

Clicking on a user presents the scorecards available for that user. The scorecard names appear to the right of the user.

Clicking on a scorecard name presents the available times for that scorecard. "Pending" times are runs that have not yet finished calculating. The pending runs are pink. "Ready" runs are runs that are finished calculating. The ready runs are green.

Clicking on the "visit" link to the right of a run will display the actual scorecard in a new tab or a window depending on your browser settings (currently not implemented - it just takes you to the public MATS right now).

### Example scorecard image

I have added an example EMC scorecard to look at [here](https://drive.google.com/drive/folders/1WFU7O5I6h-IfPN6-tWZn190b2uV1RM_d)
An interesting comparison is that what we are calling a “scorecard row” is really analogous to this entire image (minus the legend). 
Is there a better nomenclature for this?

You will be able to have multiple rows that you can expand or collapse dynamically as well as having each cell be dynamic.

You can have multiple users and multiple runs open simultaneously.

**Comment by randytpierce** [comment]
Order vertical fields variable and statistic with variable to leftmost followed by statistic. (Steve)

**Comment by bonnystrong** [comment]
I have shared a mockup of a scorecard based on my understanding from discussions at the Nov 9 meeting.  
Note that I added the concept of "column group" as well as "row group".  It seemed to me that this would be needed if you're talking about moving a set of columns around.  If not needed for development, I think it's still helpful for illustration.

**Comment by mollybsmith-noaa** [comment]
I’m a little worried about mentioning “column groups”, because the “row
group” object that a user adds in MATS will contain both row and column
fields that we’ll sort with a behind-the-scenes template. Maybe the thing
we currently refer to as “row group” should be renamed as “parameter group”
or something.

**Comment by bonnystrong** [comment]
Then how would you refer to a collection of columns that could be
rearranged as a group?

On Thu, Nov 10, 2022 at 7:12 PM Molly Smith ***@***.***>
wrote:

> I’m a little worried about mentioning “column groups”, because the “row
> group” object that a user adds in MATS will contain both row and column
> fields that we’ll sort with a behind-the-scenes template. Maybe the thing
> we currently refer to as “row group” should be renamed as “parameter group”
> or something.
>
> On Thu, Nov 10, 2022 at 7:02 PM bonnystrong ***@***.***>
> wrote:
>
> > I have shared a mockup of a scorecard based on my understanding from
> > discussions at the Nov 9 meeting.
> > Note that I added the concept of "column group" as well as "row group".
> It
> > seemed to me that this would be needed if you're talking about moving a
> set
> > of columns around. If not needed for development, I think it's still
> > helpful for illustration.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/NOAA-GSL/MATS/discussions/836#discussioncomment-4113097
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AHBWGZL4H3JZFZ62SCLL7RTWHWSLFANCNFSM6AAAAAAR2VLU54
> >
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-GSL/MATS/discussions/836#discussioncomment-4113159>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG6HZOWZCYBRHQAO24CJUNLWHWTSDANCNFSM6AAAAAAR2VLU54>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Bonny Strong
NOAA/GSL and CIRA
office: (719) 301-6195 or home: (970) 669-1188

**Comment by mollybsmith-noaa** [comment]
I mean, maybe we do want that to be a column group, and the collection of
regions, lead times, statistics, variables, etc that is currently referred
to as a “row group” should be renamed to “parameter group” or something.
